index.js

```diff
@@ -0,0 +1,162 @@
+                    pull_number: pull_number,
+                }
+            )
+        );
+
+        if (debug) {
+            console.log(
+                `Pull request: ${JSON.stringify(pull_request, undefined, 2)}`
+            );
+        }
+
+        const check_runs = unwrapResult(
+            await octokit.request(
+                "GET /repos/{owner}/{repo}/commits/{ref}/check-runs",
+                {
+                    owner: github.context.payload.repository.owner.login,
+                    repo: github.context.payload.repository.name,
+                    ref: pull_request.head.sha,
+                }
+            )
+        ).check_runs;
+
+        for (let i = 0; i < check_runs.length; i++) {
+            const check = check_runs[i];
+            if (check.conclusion === "failure") {
+                if (debug) {
+                    console.log(
+                        `Check: ${JSON.stringify(check, undefined, 2)}`
+                    );
+                }
+
+                const job = unwrapResult(
+                    await octokit.request(
+                        "GET /repos/{owner}/{repo}/actions/jobs/{job_id}",
+                        {
+                            owner: github.context.payload.repository.owner
+                                .login,
+                            repo: github.context.payload.repository.name,
+                            job_id: check.id,
+                        }
+                    )
+                );
+
+                if (debug) {
+                    console.log(`Job: ${JSON.stringify(job, undefined, 2)}`);
+                }
+
+                unwrapResult(
+                    await octokit.request(
+                        "POST /repos/{owner}/{repo}/actions/runs/{run_id}/rerun-failed-jobs",
+                        {
+                            owner: github.context.payload.repository.owner
+                                .login,
+                            repo: github.context.payload.repository.name,
+                            run_id: job.run_id,
+                        }
+                    )
+                );
+            }
+        }
+
+        core.setOutput("triggered", "true");
+
+        unwrapResult(
+            await octokit.request(
+                "POST /repos/{owner}/{repo}/issues/comments/{comment_id}/reactions",
+                {
+                    owner: owner,
+                    repo: repo,
+                    comment_id: github.context.payload.comment.id,
+                    content: "+1",
+                }
+            )
+        );
+    }
+}
+
+run().catch((err) => {
+    console.error(err);
+    core.setFailed("Unexpected error");
+});
+
+function unwrapResult(response) {
+    if (response.status === 200) {
+        return response.data;
+    }
+    if (response.status === 201) {
+        return undefined;
+    }
+    console.error(`response failed ${JSON.stringify(response, undefined, 2)}`);
```

Comment:
The console can format objects for you.
```suggestion
    console.error('response failed:', response);
```